tests/components/scene/common.py

```diff
@@ -17,3 +17,9 @@ def activate(hass, entity_id=ENTITY_MATCH_ALL):
         data[ATTR_ENTITY_ID] = entity_id
 
     hass.services.call(DOMAIN, SERVICE_TURN_ON, data)
+
+
+async def async_activate(hass, entity_id=ENTITY_MATCH_ALL):
```

Comment:
Please don't add more of these helpers. We're trying to get rid of them.

Just call the service from the tests.

Reply:
Done